### PR TITLE
[keymap] Added Filco Majestouch TKL Pegasus Hoof ISO Layout

### PR DESCRIPTION
**Pull request by nygellyndley**
## Description

Adds ISO Layout for Pegasus Hoof for Filco Majestouch

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bpiphany/pegasushoof/2015/2015.h [hunk 1]
```diff
@@ -56,6 +56,22 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         KA4, KP2, KC6,                KK6,                KC0, KM3, KD0, KA1,   KO0, KK0, KL0  \
     )
 
+#define LAYOUT_tkl_iso( \
+    KJ6,      KI4, KH4, KH2, KH6, KA7, KE6, KD2, KD4, KB4, KB7, KB6, KB0,   KC7, KC5, KA5, \
+    KJ4, KJ7, KI7, KH7, KG7, KG4, KF4, KF7, KE7, KD7, KR7, KR4, KE4, KB2,   KL4, KO4, KQ4, \
+    KJ2, KJ5, KI5, KH5, KG5, KG2, KF2, KF5, KE5, KD5, KR5, KR2, KE2,        KK4, KO7, KQ7, \
+    KI2, KJ3, KI3, KH3, KG3, KG6, KF6, KF3, KE3, KD3, KR3, KR6, KB3, KB1,                  \
+    KN2, XX2, KJ1, KI1, KH1, KG1, KG0, KF0, KF1, KE1, KD1, KR0,      KN3,        KO6,      \
```
Comment (noroadsleft) [inline keyboards/bpiphany/pegasushoof/2015/2015.h]:
```suggestion
    KN2, KI6, KJ1, KI1, KH1, KG1, KG0, KF0, KF1, KE1, KD1, KR0,      KN3,        KO6,      \
```

File: keyboards/bpiphany/pegasushoof/2015/2015.h [hunk 2]
```diff
@@ -56,6 +56,22 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+    KA4, KP2, KC6,                KK6,                KC0, KM3, KD0, KA1,   KO0, KK0, KL0  \
+    ) LAYOUT( \
+        KJ6,      KI4, KH4, KH2, KH6, KA7, KE6, KD2, KD4, KB4, KB7, KB6, KB0,   KC7, KC5, KA5, \
+        KJ4, KJ7, KI7, KH7, KG7, KG4, KF4, KF7, KE7, KD7, KR7, KR4, KE4, KB2,   KL4, KO4, KQ4, \
+        KJ2, KJ5, KI5, KH5, KG5, KG2, KF2, KF5, KE5, KD5, KR5, KR2, KE2, KB3,   KK4, KO7, KQ7, \
+        KI2, KJ3, KI3, KH3, KG3, KG6, KF6, KF3, KE3, KD3, KR3, KR6,      KB1,                  \
+        KN2, XX2 ,KJ1, KI1, KH1, KG1, KG0, KF0, KF1, KE1, KD1, KR0,      KN3,        KO6,      \
```
Comment (noroadsleft) [inline keyboards/bpiphany/pegasushoof/2015/2015.h]:
```suggestion
        KN2, KI6, KJ1, KI1, KH1, KG1, KG0, KF0, KF1, KE1, KD1, KR0,      KN3,        KO6,      \
```

File: keyboards/bpiphany/pegasushoof/2015/2015.h (hunk 1 above)
Comment (nygellyndley) [reply keyboards/bpiphany/pegasushoof/2015/2015.h]: tvm, took me an age to figure out how all this mapping was working, can't believe i left those in :(

**Comment by nygellyndley** [comment]
I added the entry to `rules.mk` as advised, the build is now failing, presumably because rules.mk contains

DEFAULT_FOLDER=bpiphany/pegasushoof/2013

I'm not sure the best way to resolve this

> Just the one thing to make the new macro consistent with the others.
> 
> Also, could you add `tkl_iso` to the `LAYOUTS` rule on Line 30 of `keyboards/bpiphany/pegasushoof/rules.mk`, please?

The `DEFAULT_FOLDER=bpiphany/pegasushoof/2013` line in the `rules.mk` file seems to be causing build issues.

Should `LAYOUTS = tkl_ansi tkl_iso` be set in the `keyboards/bpiphany/pegasushoof/2015/rules.mk` file instead?

**Comment by noroadsleft** [comment]
Oh, I thought the 2013 revision had an ISO macro already.

Revert the change to `keyboards/bpiphany/pegasushoof/rules.mk` and I can arrange it later in a separate PR.

**Comment by nygellyndley** [comment]
> 
> 
> Oh, I thought the 2013 revision had an ISO macro already.
> 
> Revert the change to `keyboards/bpiphany/pegasushoof/rules.mk` and I can arrange it later in a separate PR.

done, and added a new commit to fix the incorrect XX variable naming

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**